ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -2199,9 +2199,9 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-nutanix-ipi-image-registry-s3
-- as: nutanix-ipi-proxy-fips-f28
+- as: nutanix-ipi-proxy-fips-minimal-permission-f28
```

Comment:
Because we are covering more points in the jobs, shall we raise its frequency to `f14`?